pandas/tests/computation/test_eval.py

```diff
@@ -31,7 +31,6 @@
     assert_series_equal, makeCustomDataframe as mkdf, randbool)
 
 _series_frame_incompatible = _bool_ops_syms
```

Comment:
`_series_frame_incompatible` can also be removed. AFK for a short while. will remove.